### ISSUE TITLE
:map no longer gives output

### ISSUE THREAD

**Issue by Konfekt** [girishji/vimsuggest#17]
### Steps to reproduce

1.  vim --clean -u ~/.viminrc with `vimrc` reading 

```
filetype plugin indent on 
syntax on
let &rtp = '~/.vim/plugged/vimsuggest/' . ',' . &rtp
```

2. nmap H H<cr>
3. nmap H

### Expected behaviour

Output `nmap H H`, but no output given

### Version of Vim

9.1.902

### Environment

Tested with Ubuntu 22.04 and all combinations of Urxvt 9.30 and Xterm 372 with ZSH and Bash.

**Comment by girishji** [comment]
I followed the same steps and I get `n H     H` printed. Is this not printing to you?

**Comment by Konfekt** [comment]
No, that's what I get: https://imgur.com/BqW11QS ; I now also checked with Bash on Gnome Terminal

**Comment by girishji** [comment]
Does `:nmap` has the mapping you created? Is the mapping not getting created for any `{lhs}`? Or is it just issue with mapping `H`. I have a Mac, but I can check this out later on a gcp linux server.

**Comment by Konfekt** [comment]
Yes, it has. `H` is completely arbitrary. `L` works, any key works. If unmapped, it shows the corresponding message, but otherwise it's swallowed. How come only here?

**Comment by girishji** [comment]
I didn't parse this. 1) the mapping for `H` exists in the list printed by `nmap`, 2) `nmap H` will not print the output. 3) This bug goes away if you did not include vimsuggest into `rtp`. Is this correct?

**Comment by Konfekt** [comment]
1) Yes, it does. It is shown because of the hit Enter prompt by additional lines given by Netrw mappings 2) it will not, 3) it goes away. All correct

**Comment by girishji** [comment]
This looks like a Vim bug specific to Linux. You can workaround it by shifting the popup window one line upwards.

```
g:VimSuggestSetOptions({cmd: {popupattrs: {line: &lines - &cmdheight - 1}}})
```

**Comment by Konfekt** [comment]
Thank you, indeed that solves the issue. Should that be reported upstream then?

**Comment by girishji** [comment]
Should be. But I do not have time to create a minimal test case for reproducing the bug.
